src/Microsoft.DotNet.Build.CloudTestTasks/SendToHelix.cs

```diff
@@ -91,6 +92,11 @@ public override bool Execute()
                         Log.LogMessage(MessageImportance.High, "Started Helix job: CorrelationId = {0}", JobId);
                         return true;
                     }
+                    else
+                    {
+                        string responseContent = await response.Content.ReadAsStringAsync();
+                        Log.LogWarning(string.Format("Helix Api Response: StatusCode {response.StatusCode} {responseContent}"));
```

Comment:
I checked, this will cause a runtime error.  I thought it might be a snazzy new C# feature, not so much.